content/en/sensitive_data_scanner/_index.md

```diff
@@ -28,67 +28,121 @@ Sensitive data, such as credit card numbers, bank routing numbers, and API keys
 
-To use Sensitive Data Scanner, set up a scanning group to define what data to scan and then set up scanning rules to determine what sensitive information to match within the data. 
-
-If you want to redact your sensitive data in your environment before shipping to your downstream destinations, see how to [redact sensitive data with Observability Pipelines][14].
+To use Sensitive Data Scanner, set up a scanning group to define what data to scan and then set up scanning rules to determine what sensitive information to match within the data.
 
 This document walks you through the following:
 
-- [The permissions required to view and set up Sensitive Data Scanner](#permissions).
-- [Setting up scanning for sensitive data](#set-up-scanning-for-sensitive-data).
-- [Using the out-of-the-box dashboard](#out-of-the-box-dashboard).
+- The permissions required to view and set up Sensitive Data Scanner.
+- Setting up scanning for sensitive data.
+- Using the out-of-the-box dashboard.
 
 **Note**: See [PCI DSS Compliance][1] for information on setting up a PCI-compliant Datadog organization.
 
 {{< img src="sensitive_data_scanner/sds_main_12_01_24.png" alt="The Sensitive Data Scanner page showing six out of the 12 active scanning groups" style="width:90%;">}}
 
-## Permissions
+## Set up Sensitive Data Scanner
+<br>
+{{< callout url="https://google.com" >}}
+  Sensitive Data Scanner using the Agent is in private beta. To request access, fill out this form.
+{{< /callout >}} 
 
-By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant the `data_scanner_read` or `data_scanner_write` permissions under [Compliance][2] to a custom role. See [Access Control][3] for details on how to set up roles and permissions.
+There are two locations where you can redact your sensitive data:
 
-{{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}
+**In the cloud:**
 
+- With **Sensitive Data Scanner in the Cloud**, you submit your logs in the Datadog backend. In this method, logs leave your premises before they are redacted. You can have multiple scanning groups per organization, and you can create custom scanning rules. You can also redact sensitive data in tags.
 
-## Set up Sensitive Data Scanner
+**In your environment:**
+
+- With **Sensitive Data Scanner using the Agent**, Datadog redacts your logs before submitting them to the Datadog backend, and unredacted logs never need to leave your premises. With this method, you are limited to one scanning group per organization, and you can use only predefined library rules.
+
+   {{< img src="sensitive_data_scanner/sds_using_agent.png" alt="A diagram showing the flow of data from a customer environment to Datadog, where logs are redacted using the Agent before they leave the customer environment" style="width:80%;">}}
```

Comment:
This image does not illustrate that the data is redacted in the environment with the agent. It mostly show what is the difference between On-prem vs In-Cloud. I wonder if this image can be put before the `In the cloud` section. 
Another option is to change the names in the arrow from `Metrics, Logs, Etc.` to `LOGS` & `REDACTED LOGS` similar to the [second image](https://dd.slack.com/archives/C05AFJECCKS/p1718908957892549?thread_ts=1718893542.977289&cid=C05AFJECCKS) (and ideally removing `unprocessed` word because `unprocessed` is not opposed to `redacted`)

Reply:
I cannot edit the words in the image. I can change it to the other image if you'd like, or move the image so another place.

Reply:
Thanks for the context, I don't want to block the PR on this, I pinged @LutaoX  to see if this can be improved in a follow-up PR (or in this one if this update can be done quickly).

Reply:
Agree with @vinckama 's comment. Indeed , in the blog post, we are utilizing the currently included image to explain "on-prem" vs "cloud" but have a separate image to illustrate the "redaction activity" between OP and Agent,  see this link: 
https://drive.google.com/file/d/1FJxXDPvnEOaQa1k4DKvDD8tMyBOVedg0/view?usp=sharing

Reply:
Removing image for now.